newa/cli.py

```diff
@@ -189,6 +197,19 @@ def main(click_context: click.Context,
     if not ctx.state_dirpath.exists():
         ctx.logger.debug(f'State directory {ctx.state_dirpath} does not exist, creating...')
         ctx.state_dirpath.mkdir(parents=True)
+
+    # extract YAML files from the given archive to state-dir
+    if extract_state_dir:
+        if re.match('^http[s]?:', extract_state_dir):
```

Comment:
Out of curiosity, why `[s]?`, wouldn’t `s?` work as well?

Reply:
Just my habit. Fixed. Thank you for a review.